include/interactive_markers/interactive_marker_server.h

```diff
@@ -212,9 +212,6 @@ class InteractiveMarkerServer : boost::noncopyable
 
   // updates that have to be sent on the next publish
   M_UpdateContext pending_updates_;
-
-  // topic namespace to use
-  std::string topic_ns_;
```

Comment:
If we go forward with this PR, I would leave this member here (unused) so we maintain ABI-compatibility.